README.md

```diff
@@ -536,30 +536,70 @@ Also, you can view the performance and call drill downs in the `Performance` bla
+    - [AzureKeyVault](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-key-vault?view=azure-devops) task is used in both the stages to fetch the secrets from Azure Key Vault instance and set as variables. In the `Deploy` stage, these variables are used to set secrets in the pods.
+    - [Kubernetes Manifest task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/kubernetes-manifest?view=azure-devops) has the added benefits of being able to check for object stability before marking a task as success/failure, perform artifact substitution, add pipeline traceability-related annotations onto deployed objects, simplify creation and referencing of imagePullSecrets
+
+    ![](./media/azure-pipelines-01.jpg)
+
+3. After you've looked at what the pipeline does, select Save and run, after which you're prompted for a commit message because Azure Pipelines adds the azure-pipelines.yml file to your repository. After editing the message, select Save and run again to see your pipeline in action.
+
+4. As your pipeline runs, watch as your build stage, and then your deployment stage, go from blue (running) to green (completed). You can select the stages and jobs to watch your pipeline in action.
+
+    ![](./media/azure-pipelines-02.jpg) 
+
+5. Additionally, you can also explore Kubernetes objects created and the deployment history for the App by navigating to the "**Environment**". 
+    - From the pipeline **summary** -> Select the **Environments** tab -> Select **View environment**
+    - **Resources** view within the environment provides a glimpse of the status of objects within the namespace mapped to the resource. It also overlays pipeline traceability on top of these objects so that one can trace back from a Kubernetes object to the pipeline and then back to the commit.
 
-![](./media/azure-pipelines-01.jpg)
-![](./media/azure-pipelines-02.jpg) 
+    ![](./media/azure-pipelines-03.jpg) 
 
 ### GitHub Actions
 
-You can configure a 
+You can also configure a 
 [Workflow to automate build and deploy](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow) 
 in GitHub 
-using the [`workflow.yml`](./.github/workflows/workflow.yml). Use the steps outlined in
-[Creating and Encrypting Secrets in GitHub](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
-and create secrets for every parameter in `.scripts/setup-env-variables-azure-template.sh`.
+using the [`workflow.yml`](./.github/workflows/workflow.yml). 
+
+- Create secrets for every parameter in `.scripts/setup-env-variables-azure-template.sh`.
+    - Set [GitHub Repo Secrets](https://aka.ms/create-secrets-for-GitHub-workflows) AZURE_CREDENTIALS, AKS_CLUSTER_NAME, AKS_RESOURCE_GROUP and AKS_NAMESPACE with Azure creds, AKS cluster name, resource group name and namespace
```

Comment:
This line suggests that GitHub secrets are the only mechanism for storing secrets. However, later in the section, we are offering either GitHub Secrets OR Azure Key Vault for storing secrets. Suggest that you mention the same here.

Reply:
Fair enough. Let's include the changes suggested. Thanks